python-avd/pyavd/_eos_designs/structured_config/underlay/static_routes.py

```diff
@@ -43,6 +43,19 @@ def static_routes(self: AvdStructuredConfigUnderlay) -> list[dict] | None:
                 {"destination_address_prefix": l3_interface_static_route.prefix, "gateway": l3_interface.peer_ip}
                 for l3_interface_static_route in l3_interface.static_routes
             )
+        # similar exercise for L3 Port-Channels
+        for l3_port_channel in self.shared_utils.l3_port_channels:
```

Comment:
Could we join the two lists on the for loop above instead to avoid repeating the same code?

Reply:
Addressed this by allowing logic to act upon merged lists by using unpacking operator (*)